docs/connect.rst

```diff
@@ -0,0 +1,48 @@
+.. _schemaregistry_kafka_connect:
+
+Kafka Connect
+=============
+
+Kafka Connect and Schema Registry integrate to capture schema information from connectors. :ref:`Kafka Connect converters <connect_converters>`
+provide a mechanism for converting data from the internal data types use by Kafka Connect to data types represented as Avro.
+The AvroConverter will convert existing Avro data to the internal data types used by Kafka Connect. This will provide schema
+information to Sink Connectors.
```

Comment:
This makes it sound like it's just for sink connectors, at the exclusion of source connectors.